pkg/security/probe/selftests/windowsCreateFile.go

```diff
@@ -0,0 +1,58 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
```

Comment:
this file should be named `create_file_windows.go` and you could then remove the build tag